src/todoList.viewModel.js

```diff
@@ -81,5 +87,12 @@ export class ViewModel {
 
     clearCompletedTodos() {
         this.todoList(this.todoList().filter((todo) => !todo.checked()));
+        this.saveState();
+    }
+
+    saveState() {
+        todosRepository.save(this.todoList());
```

Comment:
It is possible to pass observable itself, not it's value. ko.toJS will get the value from it itself